code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -97,6 +97,22 @@ nobiliumsuppression = INFINITY
 			air.gases[/datum/gas/water_vapor][MOLES] -= MOLES_GAS_VISIBLE
 			. = REACTING
 
+//freon: helps the station chill out
+/datum/gas_reaction/freon_freeze
+	priority = 1
+	name = "Freon Freeze"
+	id = "freon_freeze"
+
+/datum/gas_reaction/freon_freeze/init_reqs()
+	min_requirements = list(/datum/gas/freon = MOLES_GAS_VISIBLE)
+
+/datum/gas_reaction/freon_freeze/react(datum/gas_mixture/air, datum/holder)
+	var/turf/open/location = isturf(holder) ? holder : null
+	. = NO_REACTION
+	if(location?.freon_gas_act())
+		air.temperature = max(2.7, air.temperature - 10)
+		. = REACTING
```

Comment:
So it just cools for no reason? Not even consuming the gas? Where the heat goes?

Reply:
Also we have another reaction that does almost the same thing, change that to use the sliding thing. We just removed several reactions to make the react proc lighter.

Reply:
Which reaction are you talking about?

Reply:
Freon fire